package.json

```diff
@@ -29,9 +36,12 @@
     "babelify": "^8.0.0",
     "browserify": "^16.2.3",
     "chai": "^4.1.2",
+    "cpx": "^1.5.0",
```

Comment:
Seems like this dependency can be removed since we never use it. Or did I miss something?

Reply:
It is in fact obsolete. i'll remove it.